content/shmem_finalized.tex

```diff
@@ -0,0 +1,32 @@
+\apisummary{
+  Indicates whether the \openshmem library has been finalized.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+int @\FuncDecl{shmem\_finalized}@(void);
+\end{Csynopsis}
+
+\begin{apiarguments}
+  \apiargument{None}{}{}
+\end{apiarguments}
+
+\apidescription{
+  The \FUNC{shmem\_finalized} routine returns a value indicating
+  whether the \openshmem library has been finalized (i.e, a call to
```

Comment:
Do we want "has been" or "is currently"? The problem with "has been" is that it tells you whether the API was ever called. If we assume that an OpenSHMEM library can't be reinitialized, then this is enough information to tell you the library state. If an OpenSHMEM library supports reinitialization then "has been" is not enough information to know the state of the library. You need to know "is currently".

Reply:
For reference, here is the NVSHMEM library state query function: https://docs.nvidia.com/hpc-sdk/nvshmem/api/docs/gen/api/setup.html#nvshmemx-init-status